src/pages/contact.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const RequestHelpPage = () => {
+    return (
+        <main>
+            <h1>
+                Text NEIGHBOR to <a href="tel:+19176344426">(917) 634-4426</a>
```

Comment:
Rohan said that _Product team suggests “Hi” - it's short and concise._
Please update:

```suggestion
                Text Hi to <a href="tel:+19176344426">(917) 634-4426</a>
```

Reply:
Good point. `Neighbor` is pretty long. `Hi` seems a little too cheery, no? How about `Hello`? Ultimately, it can be anything except for what it is now. `HELP` is a reserved word.

Reply:
@Miikis Did you talked with Rohan about this?

Reply:
Hello is good